install/envs/ubuntu.yml

```diff
@@ -36,6 +37,7 @@ dependencies:
   - plotly
   - pyyaml
   - tensorflow=2.2.0
+  - fastai
```

Comment:
Please also add the module in `setup.py`. Do you know if this installs on the RPi w/o problems?

Reply:
I think it should work as fastai is a wrapper around pytorch so there is no binaries as such. I will test this.